packages/lib/src/smartthings-command.ts

```diff
@@ -85,7 +85,7 @@ export abstract class SmartThingsCommand extends Command implements SmartThingsC
 	private _logger?: log4js.Logger
 	get logger(): log4js.Logger {
 		if (!this._logger) {
-			this._logger = log4js.getLogger('cli')
+			this._logger = log4js.getLogger(`cli.${this.constructor.name}`)
```

Comment:
This gives us some runtime context for free via log4js "Categories inheritance". All logs will be routed to the default category, or to the `cli` category if defined.